Resources/Locale/en-US/launcher/launcher-connecting.ftl

```diff
@@ -18,3 +18,4 @@ connecting-state-ResolvingHost = Resolving host
 connecting-state-EstablishingConnection = Establishing connection
 connecting-state-Handshake = Handshake
 connecting-state-Connected = Connected
+copy-reason = Copy Message
```

Comment:
This localization key needs to start with `connecting-`  and also `-reason` doesn't make much sense.